auth_from_http_remote_user/res_users.py

```diff
@@ -34,10 +34,10 @@ class res_users(orm.Model):
                                readonly=True),
     }
 
-    def copy(self, cr, uid, rid, defaults=None, context=None):
-        defaults = defaults or {}
-        defaults['sso_key'] = False
-        return super(res_users, self).copy(cr, uid, rid, defaults, context)
+    def copy(self, cr, uid, id, default=None, context=None):
+        default = default or {}
```

Comment:
Use here `default = {} if default is None else default` to not reinstantiate variable in case of False evaluation of it.